From schwab@suse.de Thu May 30 20:04:33 2002
Date: Wed, 29 May 2002 17:55:58 +0200
From: Andreas Schwab <schwab@suse.de>
To: binutils@sources.redhat.com
Subject: selective[1245] xfail on x86-64
Message-ID: <jesn4bymlt.fsf@sykes.suse.de>
MIME-Version: 1.0
Content-Type: text/plain; charset=iso-8859-1
Content-Transfer-Encoding: 8bit
Status: RO

The linker tests selective[1245] fail on x86-64 because of relocation
references from .eh_frame.  OK to mark xfail?

Andreas.

2002-05-29  Andreas Schwab  <schwab@suse.de>

	* ld-selective/selective.exp: Mark selective1, selective2,
	selective4 and selective5 xfail on x86_64*-*.

============================================================
Index: ld/testsuite/ld-selective/selective.exp
--- ld/testsuite/ld-selective/selective.exp	9 Jan 2004 21:19:15 -0000	1.30
+++ ld/testsuite/ld-selective/selective.exp	15 Jan 2004 07:58:34 -0000
@@ -39,11 +39,11 @@ if {[istarget "ia64-*-*"]} {
 #
 # FIXME: Instead of table, read settings from each source-file.
 set seltests {
-  {selective1 C   1.c  {}       {}    {dropme1 dropme2} {}}
-  {selective2 C   2.c  {}       {}    {foo} {mips*-*}}
+  {selective1 C   1.c  {}       {}    {dropme1 dropme2} {x86_64*-*}}
+  {selective2 C   2.c  {}       {}    {foo} {mips*-* x86_64*-*}}
   {selective3 C   2.c  {-u foo} {foo} {{foo 0}} {mips*-*}}
-  {selective4 C++ 3.cc {}       {start a A::foo() B::foo()} {A::bar()} {mips*-*}}
-  {selective5 C++ 4.cc {}       {start a A::bar()} {A::foo() B::foo()} {mips*-*}}
+  {selective4 C++ 3.cc {}       {start a A::foo() B::foo()} {A::bar()} {mips*-* x86_64*-*}}
+  {selective5 C++ 4.cc {}       {start a A::bar()} {A::foo() B::foo()} {mips*-* x86_64*-*}}
   {selective6 C++ 5.cc {}       {start a A::bar()}
     {A::foo() B::foo() dropme1() dropme2()} {*-*-*}}
 }

-- 
Andreas Schwab, SuSE Labs, schwab@suse.de
SuSE Linux AG, Deutschherrnstr. 15-19, D-90429 Nürnberg
Key fingerprint = 58CA 54C7 6D53 942B 1756  01D3 44D5 214B 8276 4ED5
"And now for something completely different."



